web/src/main/java/com/futurewei/alcor/web/entity/dataplane/InternalPortEntity.java

```diff
@@ -15,27 +15,86 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.futurewei.alcor.web.entity.port.PortEntity;
 import com.futurewei.alcor.web.entity.route.RouteEntity;
+import com.futurewei.alcor.web.entity.vpc.VpcEntity;
 import lombok.Data;
 
-import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
 
 @Data
 public class InternalPortEntity extends PortEntity {
 
-    @JsonProperty("routes")
-    private List<RouteEntity> routes;
+  @JsonProperty("routes")
```

Comment:
See some inline changes. In IntelliJ, please click Ctrl+Shift+L which will trigger reformatting of the file based on recommended Java code style.

Reply:
@haboy52581 It seems that the codes are not formatted yet. Could you take care of this? This should be quick.